hathor/cli/openapi_files/openapi_base.json

```diff
@@ -7,7 +7,7 @@
         ],
 	"info": {
```

Comment:
It seems useful to add the script that checks whether all versions are the same. @luislhl What do you think?

Reply:
Done.